protobuf/protoc_echo_plugin/test/CMakeLists.txt

```diff
@@ -1,5 +1,11 @@
+if (EMIL_FETCH_ECHO_COMPILERS)
```

Comment:
If you were to revert the logic, I think you wouldn't need this workaround

Reply:
Yeah... but then I would adjust the interface to accommodate the implementation, that feels backwards